kimchi/src/proof.rs

```diff
@@ -138,52 +144,16 @@ where
 
 //~ spec:endcode
 
-impl<F> ProofEvaluations<F> {
-    pub fn transpose<const N: usize>(
```

Comment:
decided to get rid of this as it is highly error prone and doesn't add much. We need to implement a macro to do the job.